activation/2.0/_index.md

```diff
@@ -1,35 +1,31 @@
 ---
-title: "Jakarta Activation 2.0 (under development)"
-date: 2020-mm-dd
+title: "Jakarta Activation 2.0"
+date: 2020-06-05
 summary: "Release for Jakarta EE 9"
 ---
-# Need scope statement
+Jakarta Activation defines a set of standard services to: determine the
+MIME type of an arbitrary piece of data; encapsulate access to it;
+discover the operations available on it; and instantiate the
+appropriate bean to perform the operation(s).
 
-* [Jakarta Activation 2.0 Release Record]()
+* [Jakarta Activation 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jaf/releases/2.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Activation 2.0 Specification Document]() (PDF)
-* [Jakarta Activation 2.0 Specification Document]() (HTML)
+* [Jakarta Activation 2.0 Specification Document](activation-spec-2.0.pdf) (PDF)
+* [Jakarta Activation 2.0 Specification Document](activation-spec-2.0.html) (HTML)
 * [Jakarta Activation 2.0 Javadoc](./apidocs)
-* [Jakarta Activation 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Activation 2.0 TCK](http://downloads.eclipse.org/jakarta/activation/2.0/jakarta-activation-tck-2.0.0.zip) ([sig](http://downloads.eclipse.org/jakarta/activation/2.0/jakarta-activation-tck-2.0.0.zip.sig),[sha](http://downloads.eclipse.org/jakarta/activation/2.0/jakarta-activation-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.activation:jakarta.activation-api:jar:2.0.0]()
+  * [jakarta.activation:jakarta.activation-api:jar:2.0.0](https://search.maven.org/artifact/jakarta.activation/jakarta.activation-api/2.0.0/jar)
 
 
 # Compatible Implementations
 
-* [Jakarta Activation]()
+* [Jakarta Activation](https://eclipse-ee4j.github.io/jaf/)
```

Comment:
This should be the download link or where you can easily find a downloadable link...

Reply:
they'll get there once https://github.com/eclipse-ee4j/jaf/pull/45 is merged and some jaf  project committer creates a release at github repo

Reply:
old conversation, resolving